catalog/app/components/Form/Field.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import type * as RF from 'react-final-form'
+import * as M from '@material-ui/core'
+
+const useFieldInputStyles = M.makeStyles({
+  root: {
+    // It hides M.CircularProgress (spinning square) overflow
+    overflow: 'hidden',
+  },
+})
+
+interface FieldProps {
+  error?: string
+  helperText?: React.ReactNode
+  input: RF.FieldInputProps<string>
+  validating?: boolean
+}
+
+export default function Field({
```

Comment:
"Field" seems too generic imo, especially considering that in fact it's a TextField under the hood

Reply:
Renamed to `TextField`